src/test/ui/target-feature-wrong.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// ignore-arm
+// ignore-aarch64
```

Comment:
Needs `ignore-wasm` here. Actually I think all non-x86 architectures need to be ignored.

```
[00:47:27] failures:
[00:47:27] 
[00:47:27] ---- [ui] ui/target-feature-wrong.rs stdout ----
[00:47:27] 	diff of stderr:
[00:47:27] 
[00:47:27] 28	26 | #[target_feature(enable = "sse2")]
[00:47:27] 29	   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[00:47:27] 30	
[00:47:27] +	error: the feature named `sse2` is not valid for this target
[00:47:27] +	  --> $DIR/target-feature-wrong.rs:26:18
[00:47:27] +	   |
[00:47:27] +	26 | #[target_feature(enable = "sse2")]
[00:47:27] +	   |                  ^^^^^^^^^^^^^^^
[00:47:27] +	
[00:47:27] +	error: aborting due to 5 previous errors
[00:47:27] -	error: aborting due to 4 previous errors
[00:47:27] 32	
[00:47:27] 33	
```